### PR TITLE
#4 prepare 1.0.0 release

### PR DESCRIPTION
**Pull request by SaveTheAles**

### CODE REVIEW

**Comment by SaveTheAles** [comment]
@all-contributors please add @Jonybang  for code

**Comment by SaveTheAles** [comment]
@all-contributors please add @Jonybang for code and tests and documentation

**Comment by SaveTheAles** [comment]
@all-contributors please add @Jonybang for code and tests and documentation and maintenance

**Comment by SaveTheAles** [comment]
@all-contributors please add @litvintech  for code and project management and documentation

**Comment by SaveTheAles** [comment]
@all-contributors please add @TanNgocDo for code

**Comment by SaveTheAles** [comment]
@all-contributors please add @SaveTheAles  for documentation

**Comment by SaveTheAles** [comment]
@all-contributors please add @litvintech for code and project management and documentation

**Comment by SaveTheAles** [comment]
@all-contributors please add @TanNgocDo for code

**Comment by SaveTheAles** [comment]
@all-contributors please add @SaveTheAles for documentation

**Comment by SaveTheAles** [comment]
Almost done

we need to: 
- [ ] create logo
- [x] define overview section
- [x]  define TODO section
- [x] define TOFI section 

@Jonybang @litvintech need your help

**Comment by MicrowaveDev** [comment]
TODO:
- implement Tendermint builder and RPC for generate and send transactions for general purposes that will be valid for any cosmos network without using cosmos-sdk (aka LCD, Gaia-Lite)
- implement builders and RPC for popular cosmos networks
- emulate RPC server in tests and test requests sending for RPC classes
- improve tests coverage

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                  git config --global user.name "Cyber Admin"
+                  git add CHANGELOG.md
+                  git commit -m "Circle CI: Update Changelog"
+                  git push -q https://${DOCS_GITHUB_TOKEN}@github.com/cybercongress/js-cosmos.git master
+                fi
+
+  update_doc:
+    working_directory: ~/build
+    docker:
+      - image: circleci/golang:1.12
```
Comment (Jonybang) [inline .circleci/config.yml]: i think need some javascript docker image

**Comment by SaveTheAles** [comment]
@litvintech @Jonybang 
Add an overview 
Waiting for your review

**Review by MicrowaveDev — APPROVED** [review]

**Review by cyborgshead — APPROVED** [review]

**Comment by cyborgshead** [comment]
references https://github.com/cybercongress/js-cosmos/issues/4

**Pull request merged**